pkg/handler/iam_handler.go

```diff
@@ -166,7 +169,17 @@ func updatePolicy(p *iampb.Policy, bs []*v1alpha1.Binding, expiry time.Time) {
 			result = append(result, cb)
 			continue
 		}
-		// TODO (#6): Remove expired bindings.
+
+		// Skip expired bindings.
+		expired, err := expired(cb.Condition.Expression)
```

Comment:
Since AOD is an important emergency response tool, maybe we should allow the update to continue even if the expired policies can't be removed? It would be unfortunate if we had to block an emergency response that could have continued. Maybe we should find a way to alert about the problem but still continue?

Reply:
Yeah, thanks for pointing it out, I thought about this as well. The reason I chose to stop here mainly because we don't expect this error, if there is, it probably means that the conditions are polluted and AOD is producing unexpected IAM updates, in this case someone/admin needs to take a look at this right away. I was wondering how to alert people about this problem and couldn't find a way except for using metrics (probably GMA or GCP metrics?). What is your suggestion?